### PR TITLE
launch file with urdf

### PR DESCRIPTION
**Pull request by itsgohtime**

### CODE REVIEW

File: config/luci_view.rviz [hunk 1]
```diff
@@ -0,0 +1,195 @@
+Panels:
+  - Class: rviz_common/Displays
```
Comment (Christian-Prather) [inline config/luci_view.rviz:2]: I see why this is here to allow the launch file below to access it but is there anything from this that should be upstream to the generic one we provide here https://github.com/lucimobility/luci-ros2-sdk/tree/main/rviz

File: urdf/awl_description.xacro [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  <joint name="right_main_wheel_joint" type="continuous">
+    <axis xyz="0 0 -1"/>
+    <parent link="base_link"/>
+    <child link="right_main_wheel"/>
+    <origin rpy="0 0 0" xyz="0.0 ${chair_width/2 + main_wheel_length/2} ${main_wheel_radius + 0.02}"/> 
+  </joint>
+
+  <link name="right_main_wheel">
+    <!-- <visual>
+      <geometry>
```
Comment (Christian-Prather) [inline urdf/awl_description.xacro:62]: Commented code

File: launch/awl_description.launch.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import launch
```
Comment (Christian-Prather) [inline launch/awl_description.launch.py:1]: Can you please as a README to the launch folder that briefly describes what each launch file will do.

**Review by Christian-Prather — CHANGES_REQUESTED** [review]
Looking really good a couple small changes please and this is good to go

File: config/luci_view.rviz (hunk 1 above)
Comment (itsgohtime) [reply config/luci_view.rviz:2]: The only real difference in this launch file is that we have the robot description topic already subscribed, which can be added upstream

Comment (Christian-Prather) [reply config/luci_view.rviz:2]: Okay, I dont think its a huge deal to pull the rivz changes up to the main one as the URDF stuff is not a default currently in the SDK. Im fine with how this is then. If it becomes used more often then we can upstream the RVIZ changes.

**Review by Christian-Prather — APPROVED** [review]
Approved changes look good

**Review by lupak — APPROVED** [review]

**Pull request merged**